nautobot/core/management/commands/generate_performance_test_endpoints.py

```diff
@@ -0,0 +1,228 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.urls import get_resolver
+from django.utils.http import urlencode
+import yaml
+
+from nautobot.core.utils.lookup import get_model_for_view_name
+
+
+class Command(BaseCommand):
+    """
+    Example usage: `nautobot-server generate_performance_test_endpoints > endpoints.yml`
```

Comment:
Would be good, I think, to add a unit test that runs this command, then loops through the generated data and, as a superuser, hits every identified endpoint and confirms that it returns a 200 status? Might be a bit slow though.